src/components/shoppingList/index.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import EditIcon from '@material-ui/icons/Edit';
+import DeleteIcon from '@material-ui/icons/Delete';
+import { ShoppingProps } from "../../utils/interfaces";
```

Comment:
Inconsistent quotes. Use either double quotes or single quotes (single quotes preferrably)

Reply:
Don't put props interface in utils. Put it in the same file with the component

Reply:
Interface for the component's props should be called ShoppingListProps